_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
+-
+  code: sq
+  name: Albanian
+  family: sqj
+  scripts: [ Latn, Elba ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ al ]
+
+-
+  code: sr
+  name: Serbian
+  family: ine
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - synthetic
+  variants: [ ba, me, rs ]
+
+-
+  code: st
+  name: Southern Sotho
+  family: bnt
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ za ]
+
+-
+  code: su
+  name: Sundanese
+  family: map
+  scripts: [ Latn, Sund ]
+  typology:
+    - SVO
+    -
+  variants:
+
+-
+  code: sv
+  name: Swedish
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ fi, se ]
+
+-
+  code: sw
+  name: Swahili
+  family: nic
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ cd, ke, tz, ug ]
+
+-
+  code: ta
+  name: Tamil
+  family: dra
+  scripts: [ Taml ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ in, lk, my, sg ]
+
+-
+  code: te
+  name: Telugu
+  family: dra
+  scripts: [ Telu ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ in ]
+
+-
+  code: tg
+  name: Tajik
+  family: iir
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - analytic
+  variants: [ tj ]
+
+-
+  code: th
+  name: Thai
+  family: tai
+  scripts: [ Thai ]
+  typology:
+    - SVO
+    - analytic, isolating
+  variants: [ th ]
+
+-
+  code: tk
+  name: Turkmen
+  family: trk
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ iq ]
+
+-
+  code: tl
+  name: Tagalog
+  family: map
+  scripts: [ Latn ]
+  typology:
+    - VSO
+    - agglutinative
+  variants: [ ph ]
+
 -
   code: tr
   name: Turkish
   family: trk
   scripts: [ Latn ]
-  typology: 
+  typology:
     - SOV
     - agglutinative
   variants: [ tr, cy ]
 
 -
-  code: ru
-  name: Russian
+  code: tt
+  name: Tatar
+  family: trk
+  scripts: [ Cyrl ]
+  typology:
+    - SOV
+    - agglutinative
+  variants:
+
+-
+  code: ug
+  name: Uighur
+  family: trk
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ cn ]
+
+-
+  code: uk
+  name: Ukrainian
   family: sla
   scripts: [ Cyrl ]
-  typology: 
+  typology:
     - SVO
+    - fusional, synthetic
+  variants: [ ua ]
+
+-
+  code: ur
+  name: Urdu
+  family: iir
+  scripts: [ Arab ]
+  typology:
+    - SOV
+    - fusional
+  variants: [ in, lk, pk ]
+
+-
+  code: uz
+  name: Uzbek
+  family: trk
+  scripts: [ Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ uz ]
+
+-
+  code: vi
+  name: Vietnamese
+  family: aav
+  scripts: [ Hani, Latn ]
+  typology:
+    - SVO
+    - analytic, isolating
+  variants: [ vn ]
+
+-
+  code: xh
+  name: Xhosa
+  family: nic
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ za ]
+
+-
+  code: yi
+  name: Yiddish
+  family: gem
+  scripts: [ Hebr ]
+  typology:
+    - V2
     - inflected
-  variants: [ ru, md, kz, ua ]
+  variants: [ il ]
```

Comment:
I'd remove this, the variants don't really correspond to the current location of speakers.